### ISSUE TITLE
Edits by "mandyboy"

### ISSUE THREAD

**Issue by TagaSanPedroAko** [OSMPH/papercut_fix#32]
This seems to be a new user, but there are problems regarding his edits. This user has deleted many significant features around the Molino area in Bacoor, and the Ecological Park (and St. Jerome Emiliani Institute?) are now gone. Please keep an eye for his edits, and revert any changeset that are deemed harmful because it is a mass deletion without a detailed explanation or rationale. The edits by this user are marked for review, so we can catch up with those if we can.

**Comment by govvin** [comment]
Helpful references

- [mandyboy](https://api.openstreetmap.org/api/0.6/changesets?user=1009565) registered in 2012 but doesn't appear to have much experience yet.
- [HDYC](http://hdyc.neis-one.org/?mandyboy) profile
- Several of their edits has been flagged in [OSMCha](https://osmcha.mapbox.com/?filters=%7B%22in_bbox%22%3A%5B%7B%22label%22%3A%22115.1116%2C1.2948%2C135.3850%2C21.0763%22%2C%22value%22%3A%22115.1116%2C1.2948%2C135.3850%2C21.0763%22%7D%5D%2C%22users%22%3A%5B%7B%22label%22%3A%22mandyboy%22%2C%22value%22%3A%22mandyboy%22%7D%5D%7D)  for mass deletions, with some changesets self-tagged with a review request.

@TagaSanPedroAko  you may use OSMCha to tag the harmful changests, should you decide to review their edits.

**Comment by TagaSanPedroAko** [comment]
I reverted the deletion of St. Jerome Emiliani Institute this time, but if you can catch up, you may revert the others. Fortunately, he tagged his own edits for review, so we can take action for each.

**Comment by TagaSanPedroAko** [comment]
Just spotted this user removing De La Salle Zobel Vermosa and a nearby subdivision. I have just reverted those (See http://www.openstreetmap.org/changeset/53682612, http://www.openstreetmap.org/changeset/53682450). Please keep this issue open and revert the remaining edits involving mass deletions.

**Comment by govvin** [comment]
Have you been in touch with them? Have you left comments on their changesets?

After doing so, kindly consider filing a report to [DWG](mailto:data@osmfoundation.org), and request for a temporary ban to get their attention about their counter-productive edits.

**Comment by TagaSanPedroAko** [comment]
I have added comments to changesets where I spotted several important POI's being deleted.

I prefer just a reverted of all of that user's mass deletions, but a block (at least one week) may be requested while we will restore those deleted features. The user is apparently new, and he must be told about his mass deletions and the use of a value comment to hide those data removals.

**Comment by TagaSanPedroAko** [comment]
Just warned again that user, that he may be reported to the DWG. I just spotted that user deleting residential area polygons for no reason. Even Versailles along Daang Hari has been targets by those needless mass deletions. Those are enough to warrant a block from the DWG, as we can consider those as disruptive. We may start reverting several changesets that deleted many important map data, even before we can report that user.

**Comment by TagaSanPedroAko** [comment]
I have just mailed the DWG regarding "mandyboy"'s mass deletions. We may start reverting at least part of his changesets which involved mass deletions, while the case is being decided upon.

**Comment by TagaSanPedroAko** [comment]
Have no longer seen activity by that user,  but we may start cleanup of that user's edits.

**Comment by govvin** [comment]
@TagaSanPedroAko, I have no objections to the revert action.  

Please proceed with caution with those edits that's been corrected or fixed by other users after this user made the changes.

**Comment by govvin** [comment]
@TagaSanPedroAko , I have no objections to the proposed revert action.

Please proceed with caution with subsequent edits of Mandyboy's changes.

**Comment by TagaSanPedroAko** [comment]
I have already done some reverts, but only for a few POIs, since I have edits limited to iD. So, I'm okay if you handle the rest using JOSM.

**Comment by TagaSanPedroAko** [comment]
Did two mass reverts last Sunday, see [this](https://www.openstreetmap.org/changeset/57302537#map=13/14.4349/120.9952) and [this](https://www.openstreetmap.org/changeset/57302432#map=15/14.4283/120.9957). Following this up at about Thursday and Holy Week.

**Comment by govvin** [comment]
@TagaSanPedroAko thank you for that.

It would help if somebody can post the list of change sets that needs to be reverted, and we can work collaboratively to complete any outstanding changesets.

**Comment by govvin** [comment]
@TagaSanPedroAko 

I've updated the original post to include the user's changesets.

**Comment by TagaSanPedroAko** [comment]
Made several reverts last Thursday, but cannot revert some because of pre-condition violations.

**Comment by govvin** [comment]
@TagaSanPedroAko
Please mark the change sets you've worked on.

**Comment by govvin** [comment]
This concern has been reported to DWG, and they've agreed to help with the reverts. FYI.
